assets/css/style.css

```diff
@@ -0,0 +1,220 @@
+/* The main colours I will be working with for my webpage  */
+:root {
+  --tea-green: #def2c8ff;
+  --tea-green-2: #c5dac1ff;
+  --opal: #bcd0c7ff;
+  --ash-gray: #a9b2acff;
+  --middle-grey: #898980ff;
+}
+
```

Comment:
I love the use of root variables and your colours were great. Good job with this.